### ISSUE TITLE
Gettext params does not work inside multiline tags

### ISSUE THREAD

**Issue by jusuff** [zikula/core#579]
Zikula_View::z_prefilter_gettext_params translates all params prefixed with double underscore (eg "__alt") using gt modifier.

But this does not work if tag is split to two or more lines.
For example - this will work:

```
{img src='logo_with_title.gif' modname='core' __alt='ok'}
```

but this wont:

```
{img 
    src='logo_with_title.gif' 
    modname='core' 
    __alt='ok'
}
```

This can be probably easy fixed by adding "s" flag for prefilter regexp.
So line 2992 in Zikula_View should be changed from:

```
return preg_replace('#((?:(?<!\{)\{(?!\{)(?:\s*)|\G)(?:.+?))__([a-zA-Z0-9][a-zA-Z_0-9]*=([\'"])(?:\\\\?+.)*?\3)#', '$1$2|gt:\$zikula_view', $tpl_source);
```

to:

```
return preg_replace('#((?:(?<!\{)\{(?!\{)(?:\s*)|\G)(?:.+?))__([a-zA-Z0-9][a-zA-Z_0-9]*=([\'"])(?:\\\\?+.)*?\3)#s', '$1$2|gt:\$zikula_view', $tpl_source);
```

I'm not making PR because I'm not sure if multiline tags should be supported.
Basically - they work. Smarty interprets them correctly. I've found only this issue with gettext params.
And this could save some time in future for someone, who (like me) likes to write long tags in few lines and founds gettext params broken :)

What do you think?
